Misc/NEWS.d/next/Tests/2019-05-04-21-25-19.bpo-36782.h3oPIb.rst

```diff
@@ -0,0 +1,2 @@
+Created a test C API wrapper for PyDate_FromDate and added a test for PyDate_FromDate. 
```

Comment:
@pganssle Could you help @SimiCode for the blurb entry? ```Add tests for the C-API wrapper of PyDate_FromDate ```? I am not a native speaker.

Reply:
Talked to Edison in the room, he will have an updated blurb that I have looked at. The current one is grammatically fine but could be tightened up and could use RST directives.